recipe/meta.yaml

```diff
@@ -35,13 +37,19 @@ requirements:
 test:
   imports:
     - zeep
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: http://www.python-zeep.org
```

Comment:
Also we do seem to have some issues opening the `home url` and `doc_url` as an `https` we might want to specify in a comment that `https` is not supported for the given urls at this point.

Reply:
added the comments